Dockerfile.build

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
+RUN apt-get update
+RUN apt-get install -y golang git-core libgpgme11-dev
+ENV GOPATH=/
+WORKDIR /src/github.com/projectatomic/skopeo
```

Comment:
AFAICT the question here is:

- Will we have a noticeable amount of users building from scratch using this Makefile (as opposed to using a distribution which builds however it wants)
- Do we care enough what build toolchain they use when/if they come asking questions or asking for help?

and my answers for both are, at least at the moment, “not enough to worry about”; so if @duglin wants to use Ubuntu for this, let’s.